src/App.tsx

```diff
@@ -86,12 +87,14 @@ function App({url}: AppProps) {
                         VideoPopoverMenuContextProvider,
                     ]}
                 >
-                    <CustomStatusBarAndBackground />
-                    <ErrorBoundary errorMessage="NewExpensify crash caught by error boundary">
-                        <ColorSchemeWrapper>
-                            <Expensify />
-                        </ColorSchemeWrapper>
-                    </ErrorBoundary>
+                    <KeyboardProvider>
```

Comment:
We can add this provider to the list of providers in array above (to `ComposeProviders` component)

Reply:
👍🏻